test/spec/unit/adapters/analytics/AnalyticsAdapter_spec.js

```diff
@@ -29,7 +29,6 @@ FEATURE: Analytics Adapters API
         const adapter = new AnalyticsAdapter(config);
         var spyTestGlobal = sinon.spy(window, config.global);
 
-        adaptermanager.registerAnalyticsAdapter(adapter, 'adapter');
```

Comment:
I'm sure it's all good. I was just pointing out my lack of extensive Prebid knowledge to know much about that is happening here.